src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -67,3 +72,14 @@ def dss_deletion_subscription_query(self, prefix: str) -> JSON:
 
     def dss_subscription_query(self, prefix: str) -> JSON:
         return {}
+
+    def drs_path(self, manifest_entry: JSON, metadata: JSON) -> str:
+        return '_'.join(['v1', manifest_entry['target_id'], manifest_entry['uuid']])
```

Comment:
```suggestion
        return '_'.join(('v1', manifest_entry['target_id'], manifest_entry['uuid']))
```

Reply:
Personally I find that the square brackets add clarity.